### PR TITLE
docs: add install instructions

### PR DESCRIPTION
**Pull request by beeme1mr**
Added install information to the top of the readme. I reorganized the page a bit but more still needs to be done. I'll create a new issue for that.

I've also moved the systemd and architecture information but didn't include a link to them. I would like some feedback on how we can best organize this page.

### CODE REVIEW

File: docs/systemd-service.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Run flagD as a systemd Service
```
Comment (toddbaert) [inline docs/systemd-service.md:1]: I think we might want at least a link to this from the main doc.

File: README.md [hunk 2]
```diff
@@ -5,119 +5,161 @@
...
+### Docker
+
+1. `docker pull ghcr.io/open-feature/flagd:latest`
 
 ## Example usage
 
-1. Generate the prerequisites `make generate`
-2. Build the flagd binary: `make build`
-3. Start the process: `./flagd start -f config/samples/example_flags.json --service-provider http --sync-provider filepath`
+1. Download sample flag configurations: `curl https://raw.githubusercontent.com/open-feature/flagd/main/config/samples/example_flags.json -o example_flags.json`
```
Comment (toddbaert) [inline README.md:23]: good addition.

File: README.md [hunk 3]
```diff
@@ -5,119 +5,161 @@
 ![version](https://img.shields.io/badge/version-pre--alpha-green)
 ![status](https://img.shields.io/badge/status-not--for--production-red)
 
-Flagd is a simple command line tool for fetching and presenting feature flags to services. It is designed to conform to OpenFeature schema for flag definitions.
+Flagd is a simple command line tool for fetching and presenting feature flags to services. It is designed to conform with the OpenFeature specification.
```
Comment (toddbaert) [inline README.md:8]:
```suggestion
Flagd is a simple command line tool for fetching and evaluating feature flags for services. It is designed to conform with the OpenFeature specification.
```

**Review by toddbaert — APPROVED** [review]

File: docs/systemd-service.md (hunk 1 above)
Comment (beeme1mr) [reply docs/systemd-service.md:1]: Done

**Review by AlexsJones — APPROVED** [review]

**Pull request merged**